### PR TITLE
Update qwiklabs.yaml

### PR DESCRIPTION
**Pull request by Nakasz**

### CODE REVIEW

File: labs/[1736]_configure-a-firewall-and-a-startup-script-with-deployment-manager/qwiklabs.yaml [hunk 1]
```diff
@@ -1 +1 @@
-# delete firewall rule (keep line 15 - 22)
\ No newline at end of file
+# delete firewall rule (keep line 24 - 34)
```
Comment (elmoallistair) [inline labs/[1736]_configure-a-firewall-and-a-startup-script-with-deployment-manager/qwiklabs.yaml:1]: line 24-34 its the firewall rule, they need to be deleted (?)

**Review by elmoallistair — COMMENTED** [review]
have you tried it and succeeded? last time we try it, we need to delete firewall rule

File: labs/[1736]_configure-a-firewall-and-a-startup-script-with-deployment-manager/qwiklabs.yaml (hunk 1 above)
Comment (Nakasz) [reply labs/[1736]_configure-a-firewall-and-a-startup-script-with-deployment-manager/qwiklabs.yaml:1]: yes

Comment (Nakasz) [reply labs/[1736]_configure-a-firewall-and-a-startup-script-with-deployment-manager/qwiklabs.yaml:1]: line 15 - 22 saya hapus 
okas2
okas1
okas


ERROR: (gcloud.deployment-manager.deployments.create) ResponseError: code=412, message=Configuration has invalid format: Unable to parse YAML.

Comment (Nakasz) [reply labs/[1736]_configure-a-firewall-and-a-startup-script-with-deployment-manager/qwiklabs.yaml:1]: Saya hapus line 24 - 34 yang merupakan firewall rule tidak terjadi error dan berhasil  
okas5
okas0
ikas1

**Review by elmoallistair — APPROVED** [review]

File: labs/[1736]_configure-a-firewall-and-a-startup-script-with-deployment-manager/qwiklabs.yaml (hunk 1 above)
Comment (elmoallistair) [reply labs/[1736]_configure-a-firewall-and-a-startup-script-with-deployment-manager/qwiklabs.yaml:1]: sebentar,,. yang dihapus firewall rule kan?
line 15-24 tetap di keep

Comment (elmoallistair) [reply labs/[1736]_configure-a-firewall-and-a-startup-script-with-deployment-manager/qwiklabs.yaml:1]: mungkin isi `qwiklabs.yaml` bisa diubah jadi

```
imports:
- path: qwiklabs,jinja

resources:
- name: qwiklabs
  type: qwiklabs.jinja
  properties:
    zone: us-central1-a
```

**Review by elmoallistair — CHANGES_REQUESTED** [review]
edit `qwiklabs.yaml`

```
imports:
- path: qwiklabs,jinja

resources:
- name: qwiklabs
  type: qwiklabs.jinja
  properties:
    zone: us-central1-a
```

**Comment by elmoallistair** [comment]
i'll merge it and change it myself, thanks

**Pull request merged**